src/rgw/rgw_common.h

```diff
@@ -1197,6 +1197,24 @@ inline ostream& operator<<(ostream& out, const RGWBucketIndexType &index_type)
   }
 }
 
+enum RGWBucketDataLayoutType {
+  RGWDLType_Normal = 0,
+  RGWDLType_Splitted = 1,
+};
+
+inline ostream& operator<<(ostream& out, const RGWBucketDataLayoutType &data_layout_type) 
+{
+  switch (data_layout_type) {
+    case RGWDLType_Normal:
```

Comment:
I don't think "normal" and "splitted" are the right names for these cases, if only because for the latter you'd write "split."  Maybe single-pool and split-pool?  There still hasn't been organized discussion in RGW standups, that would be helpful.